lambdas/template.yaml

```diff
@@ -17,7 +17,7 @@ Resources:
     Type: AWS::Serverless::Function
     Properties:
       CodeUri: CreateItemFunction/
-      Handler: index.handler
+      Handler: src/CreateItemFunction/index.handler
```

Comment:
Shouldn't this be taking the `CodeUri` as a base path for the `Handler` reference?